atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -150,7 +150,9 @@ private long grabLocksAndRun(SweepLocks locks) throws InterruptedException {
                 SweepBatchConfig lastBatchConfig = getAdjustedBatchConfig();
                 log.warn("The background sweep job failed unexpectedly with batch config {}."
```

Comment:
doesn't this line and the number of `{}` need to have an equivalent change?

Reply:
Yep, of course. I think I got too excited when I saw that the json params were logged correctly...